### PR TITLE
TS_function Read TDD

### PR DESCRIPTION
**Pull request by ehju**
TS_function read 함수에 대한 unittest 와 구현입니다.

### CODE REVIEW

**Review by sh1lee — APPROVED** [review]
좋아 보입니다. 나중에 Exception관련하여 협의가 필요해 보입니다.

**Review by liebeguso — APPROVED** [review]
향후 TS에서 전달한 invalid argument들은 commandParser class에 의해서 invalid처리될 것입니다. 즉, read,write를 호출하는 단계에서는 valid한 argument만 존재하게 될것 같아요.

File: TestShell_Excutor/TS_function_test.cpp [hunk 1]
```diff
@@ -9,4 +9,46 @@ class MockSSD : public DummySSD {
...
+	MockSSD ssd;
+	TS_function shell{ &ssd };
+	int lba = -1;
+
+	EXPECT_THROW(shell.read(lba), std::exception);
+}
+TEST(TS_function, ReadMockTestLbaException2) {
+	MockSSD ssd;
+	TS_function shell{ &ssd };
+	int lba = 100;
```
Comment (hh1012) [inline TestShell_Excutor/TS_function_test.cpp:36]: edge case에 대한 검증을 하는 훌륭한 UT 같습니다.

File: TestShell_Excutor/TS_function_test.cpp [hunk 2]
```diff
@@ -9,4 +9,46 @@ class MockSSD : public DummySSD {
...
+	TS_function shell{ &ssd };
+	int lba = 10;
+
+	EXPECT_CALL(ssd, read(lba))
+		.Times(1)
+		.WillOnce(testing::Return(1000));
+
+	unsigned int ret = shell.read(lba);
+
+	EXPECT_EQ(ret, 1000);
```
Comment (hh1012) [inline TestShell_Excutor/TS_function_test.cpp:53]: shell.read(lba)를 inline하여도 좋을것 같습니다.
